infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -177,6 +178,8 @@
                     + "              Metrics provider server port [default: 9999].\n             "
                     + " --snapshot-batch=<batch-size>                                            "
                     + "              Snapshot (Full) Sync batch size (number of entries)\n       "
+                    + " --max-data-message-size=<msg-size>                                       "
+                    + "              The max size of replication data message in bytes.\n   "
```

Comment:
as it is specifically for replication, can we rename it to max-replication-data-message-size?

Reply:
Sure, done.